content/python/concepts/built-in-functions/terms/globals/globals.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'globals()'
+Description: 'Returns a dictionary with all the global variables and symbols for the current program.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Dictionary'
+  - 'Scope'
+  - 'Symbol'
+  - 'Values'
+  - 'Variable Types'
+  - 'Variables'
+ CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The built-in **globals()** function allows you to access the global scope's name table, which is a writable dictionary containing the current global names and their corresponding values in your code. You can also use this function to access or even modify the value of a global variable from with functions.
+
+## Syntax
+
+```shell
+globals()
+```
+
+The `globals()` method doesn't take any parameters.
+
+The globals() method returns the dictionary of the current global symbol table.
+<br><br>
+## Examples
+
+### Example 1
+
+```py
+length = 123
+
+print(globals())
+
+```
+
+This code returns the following dictionary. <br>Notice that the **length** global variable is listed in the dictionary.
+
+```shell
+{'__name__': '__main__', '__doc__': None, '__package__': None, '__loader__': <class '_frozen_importlib.BuiltinImporter'>, '__spec__': None, '__annotations__': {}, '__builtins__': <module 'builtins' (built-in)>, 'age': 23}
+
+```
+### Example 2
+
+```py
+length = 123
+
+globals()['length'] = 125
+print('The length is:', length)
+
+```
+
+The global variable **length** is set to **123**. The global variable in this example can be modified using the **globals()** method with the dictionary key [length]. It can be also modified from within a function.
```

Comment:
```suggestion
The global variable `length` is set to `123`. The global variable in this example can be modified using the `globals()` function with the dictionary key `[length]`. It can be also modified from within a function.
```